packages/transforms/federation/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@graphql-mesh/transform-federation",
+    "version": "0.0.20",
+    "main": "dist/index.js",
+    "license": "MIT",
+    "scripts": {
+      "test": "jest --config=../../../jest.config.js",
+      "build": "tsc"
+    },
+    "peerDependencies": {
+      "graphql": "*",
+      "@apollo/federation": "*"
+    },
+    "devDependencies": {
+      "@types/jest": "25.1.4",
+      "@types/node": "13.9.2",
+      "@apollo/federation": "0.13.2",
```

Comment:
Not needed as dependency? just as dev?

Reply:
It is a peer dependency. I didn't want to force people to use our version. They might choose another. As same as, mongoose and grpc.